Zajecia/src/main/java/pl/isa/javasmugglers/MainMenu.java

```diff
@@ -48,12 +49,20 @@ public void userSelection(int userSelection) {
                 newUser.addUser();
                 break;
             case 2:
-                //Tutaj wywołują swoje metody Mikołaj i Patryk
+
                 System.out.print("Wybrano 2");
+                UserLogin userLogin1 = new UserLogin();
```

Comment:
Przydałoby się to przenieść zawartość case'ów do osobnych funkcji, żeby skrócić całego switcha.

Reply:
Zrobione